newIDE/app/src/UI/SearchBar.js

```diff
@@ -96,141 +109,266 @@ const getStyles = (props: Props, state: State) => {
+      if (reason === 'reset') {
+        // Happens when user selects an option
+        setValue('');
+        setAutocompleteValue('');
+      } else {
+        setValue(newValue);
+      }
+    };
 
-  render() {
-    const styles = getStyles(this.props, this.state);
-    const { value } = this.state;
-    const { disabled, style, buildTagsMenuTemplate, helpPagePath } = this.props;
+    const handleCancel = () => {
+      changeValue('');
+    };
+
+    const handleKeyPressed = (event: SyntheticKeyboardEvent<>) => {
+      if (shouldValidate(event)) {
+        onRequestSearch(value);
+      }
+    };
 
     return (
-      <ThemeConsumer>
-        {muiTheme => (
-          <Paper
-            style={{
-              backgroundColor: muiTheme.searchBar.backgroundColor,
-              ...styles.root,
-              ...style,
-            }}
-            square
-            elevation={1}
-          >
-            <div style={styles.searchContainer}>
-              <TextField
-                margin="none"
-                hintText={this.props.placeholder || t`Search`}
-                onBlur={this.handleBlur}
-                value={value}
-                onChange={this.handleInput}
-                onKeyUp={this.handleKeyPressed}
-                onFocus={this.handleFocus}
-                fullWidth
-                style={styles.input}
-                underlineShow={false}
-                disabled={disabled}
-                ref={this._textField}
-              />
-            </div>
-            {buildTagsMenuTemplate && (
-              <ElementWithMenu
-                element={
-                  <IconButton
-                    style={styles.iconButtonFilter.style}
-                    disabled={disabled}
-                    size="small"
+      <I18n>
+        {({ i18n }) => (
+          <ThemeConsumer>
+            {muiTheme => (
+              <Column>
+                <Line>
+                  <Paper
+                    style={{
+                      backgroundColor: muiTheme.searchBar.backgroundColor,
+                      ...styles.root,
+                      ...style,
+                    }}
+                    square
+                    elevation={1}
                   >
-                    <FilterList />
-                  </IconButton>
-                }
-                buildMenuTemplate={buildTagsMenuTemplate}
-              />
-            )}
-            {helpPagePath && (
-              <HelpIcon
-                disabled={disabled}
-                helpPagePath={helpPagePath}
-                style={styles.iconButtonHelp.style}
-                size="small"
-              />
+                    <div style={styles.searchContainer}>
+                      {tags ? (
+                        <Autocomplete
+                          options={tags}
+                          groupBy={options => i18n._(t`Apply a filter`)}
+                          classes={autocompleteStyles}
+                          freeSolo
```

Comment:
doesn't free solo mean you can enter anything even if not in the options? Do we want that?

Reply:
I does mean that. It's used for the search text. If you press "enter" it will search for this text with the selected tags/filters applied.